### DISCUSSION TITLE
Get point cloud objects for Radar using Cpp Client

### DISCUSSION THREAD

**Discussion by shauryaneel** [carla-simulator/carla#6021] [Q&A]
Hello all,

I am trying to get the point cloud objects for radar sensor using C++ Client.

What is the procedure to retrieve the pointcloud objects for radar sensor using C++ Client ?

1. Via **RadarMeasurement.h**
2. Via  **virtual void Listen(CallbackFunctionType callback) = 0;**

In python it is easy and there are lot of references which I can use.

How python has references for this, do we also have reference for C++. It will be helpful if someone can answer this or provide some references.

**Comment by shauryaneel** [comment]
Hello,

I was able to get the point cloud objects from radar.

Regards
Shaurya

**Reply by Amr-Khaled-BS** [reply]
@shauryaneel can you share how you managed to do this?
